assets/app/views/builds.html

```diff
@@ -8,21 +8,86 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div style="margin-bottom: 10px;" ng-repeat="build in builds">
-        <h3>{{build.metadata.labels.buildconfig}} <span class="small">({{build.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>            
-        <div>Status: {{build.status}}</div>
-        <div>Build strategy: {{build.parameters.strategy.type}}</div>
-        <div ng-if="build.parameters.strategy.type == 'STI'">
-          Builder image: {{build.parameters.strategy.stiStrategy.builderImage}}
+      <div ng-repeat="(buildConfigName, buildConfig) in buildConfigs">
+        <h2>{{buildConfigName}}</h2>
+        <div>Build strategy: {{buildConfig.parameters.strategy.type}}</div>
+        <div ng-if="buildConfig.parameters.strategy.type == 'STI'">
+          Builder image: {{buildConfig.parameters.strategy.stiStrategy.builderImage}}
```

Comment:
If it's optional I don't show the row when its not there, if its not optional then i expect the value to be set